SPECS/python3/python3.spec

```diff
@@ -235,8 +235,8 @@ find %{buildroot}%{_libdir} -name '*.o' -delete
 rm %{buildroot}%{_bindir}/2to3
 rm -rf %{buildroot}%{_bindir}/__pycache__
 
-# %check
-# make  %{?_smp_mflags} test
+%check
+python3 -m test
```

Comment:
Let's make sure this is using `python3` from the buildroot, and not the one installed in the pkggen worker